cpp/oneapi/dal/detail/policy.hpp

```diff
@@ -61,10 +62,14 @@ inline constexpr bool is_data_parallel_policy_v = is_data_parallel_policy<T>::va
 
 enum class cpu_extension : uint64_t {
     none = 0U,
+    #ifdef __ARM_ARCH
+    sve = 1U << 1,
```

Comment:
As this is the only architecture, should this be a shift by zero?

Reply:
since NEON would be supported in the future, SVE was assigned 2 on the list.

Reply:
@keeranroth is correct. SVE can have a zero shift and NEON - a one shift. No need to do gaps between the shifts. Newer ARM architectures will have bigger shifts.
In previous versions of oneDAL we had more architectures supported: sse2 - 0, ssse3 - 1, sse42 - 2, ...
But recently some of them were removed to reduce the build time and binary size. That's why there are those gaps between the shifts now.